app/models/submission.rb

```diff
@@ -64,6 +64,7 @@ class Submission < ApplicationRecord
     else
       none
     end
+    by_user_role.where(deleted_at: nil)
```

Comment:
added filtering here for this cope that is used to give users access to submissions based on their role